helm/README.md

```diff
@@ -159,37 +181,75 @@ You can find a full `values.yaml` with all of the optional values [here](https:/
 | `ingress.tlsEnabled`                                              | true                       | Enable TLS for Ingress Controller                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
 | `ingress.tlsSecretName`                                           | fiftyone-teams-tls-secret  | TLS Secret for certificate with all three DNS Names                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | `namespace.name`                                                  | fiftyone-teams             | Kubernetes Namespace already created for FiftyOne Teams                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| `pluginsSettings.affinity`                                        | None                       | FiftyOne Plugins service affinity rules                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
wow there are a lot of plugin setting configs. Are there plugin docs that mention these?

Reply:
In the Helm chart the "possible" settings are the same for `pluginsSettings` as they are for `appSettings` and (for the most part) `apiSettings` and `teamsAppSettings`

You can actually launch a cluster with very few of these... 

The Plugins docs wouldn't mention the system configuration for the `teams-plugins` deployments, these are just for infrastructure...